product_docs/docs/migration_portal/3.5.0/04_mp_migrating_database/01_mp_schema_extraction.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: "Performing a schema extraction"
+
+
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-migration-portal/user-guides/user-guide/3.0.1/mp_schema_extraction.html"
+---
+
+<div id="mp_schema_extraction" class="registered_link"></div>
+
+You can use the Oracle’s Data Pump utility to extract metadata from your Oracle database to an SQL dump file. Migration Portal requires this file to be uploaded to assess the extracted schemas.
+
+
+!!! Note
+    Migration Portal might become unresponsive for very large SQL files, depending on your system’s and browser’s resource availability. Migration Portal supports SQL files upto 1 GB. If your SQL file size exceeds 1 GB, extract fewer schemas at a time to reduce the SQL file size.
+
+## Extracting schemas to an SQL dump file
+
+Note that Migration Portal only requires the metadata in the SQL dump file to assess the extracted schemas.
+
+
+#### Prerequisites
+
+* If you plan on exporting schemas that are not your own, ensure that you are assigned the *DATAPUMP_IMP_FULL_DATABASE* role. Otherwise, you can only export your own schema.
```

Comment:
If this is a db role use code formatting, not italics.

Reply:
Done.